### PR TITLE
add STU3 examples to hooks

### LINKED ISSUE #293: Hook pages should include example STU3 resources (was May 2018 Ballot Comment 87)
## May 2018 Ballot Comment 87

  Submitted by @cmoesel from MITRE

  **Chapter:** Hooks
  **Section:** medication-prescribe
  **Type:** A-S :bulb:
  **In Person Requested?** No
  
  **Existing Wording:**
  > Example (DSTU2)
  
  **Comment:**
  According to Hooks documentation, hooks should provide examples across all versions of FHIR where differences are possible. Since it is a MedicationRequest (not MedicationOrder) in FHIR STU3, a FHIR STU3 example should be provided.
  
  ---
  _This issue was imported by @cds-hooks-bot from the [consolidated CDS Hooks May 2018 ballot spreadsheet](https://bit.ly/cds-hooks-may-2018-ballot-comments)._

### PR DESCRIPTION
**Pull request by isaacvetter**
add STU3 example to med-prescribe

Also change patient id in MedicationOrder resource to be consistent with CDS Hooks request wrapper.

Fixes #293 

FHIR Med Order and MedRequest resources stolen from: 
- https://r3.smarthealthit.org/MedicationRequest/smart-MedicationRequest-103?_format=json
- https://r3.smarthealthit.org/MedicationRequest/smart-MedicationRequest-104?_format=json

### CODE REVIEW

File: docs/hooks/medication-prescribe.md [hunk 1]
```diff
@@ -19,131 +19,279 @@ Field | Optionality | Prefetch Token | Type | Description
...
-              "system":"http://unitsofmeasure.org",
-              "code":"mL"
+          }
+        },
+        {
+          "resource":{
+            "resourceType":"MedicationRequest",
+            "id":"smart-MedicationRequest-104",
+            "meta":{
+              "lastUpdated":"2018-04-30T13:26:48.124-04:00",
```
Comment (kpshek) [inline docs/hooks/medication-prescribe.md]: This trailing comma makes the JSON malformed. You can test this by pasting the JSON into a linter/formatter like https://jsonformatter.curiousconcept.com/

File: docs/hooks/order-review.md [hunk 2]
```diff
@@ -19,6 +19,186 @@ Field | Optionality | Prefetch Token | Type | Description
...
+            "id":"pureeddiet-simple",
+            "identifier":[
+              {
+                "system":"http://goodhealthhospital.org/nutrition-requests",
+                "value":"123"
+              }
+            ],
+            "status":"draft",
+            "patient":{
+              "reference":"Patient/1288992",
```
Comment (kpshek) [inline docs/hooks/order-review.md]: This trailing comma needs to be removed in order for the JSON to be compliant.

Comment (isaacvetter) [reply docs/hooks/order-review.md]: Thanks, kdog! Fixed.

I only saw the one comma. 

What do you think?

Comment (kpshek) [reply docs/hooks/order-review.md]: @isaacvetter - The second trailing comma is in order-review.md on line 44:

```"reference":"Patient/1288992",```

The fix you made to medication-prescribe.md is good!

**Comment by kpshek** [comment]
Hey @isaacvetter - that trailing comma is now gone so 🎉 

However, there are a couple of new changes with the STU3 example:
- The very first opening `{` bracket was removed which makes the JSON fail to parse.
- All indentation was replaced by tabs which makes the JSON hard to read. For instance, compare the DSTU2 example vs the STU3 example here: https://github.com/cds-hooks/docs/blob/523cb0fa7e5815358d743792089dba40616778a2/docs/hooks/order-review.md

**Pull request merged**